mailpoet/tests/acceptance/EmailEditor/CreateAndSendEmailUsingGutenbergCest.php

```diff
@@ -7,7 +7,10 @@
 use MailPoet\Test\DataFactories\Settings;
 
 class CreateAndSendEmailUsingGutenbergCest {
-  public function createAndSendStandardNewsletter(\AcceptanceTester $i) {
+  public function createAndSendStandardNewsletter(\AcceptanceTester $i, $scenario) {
+    if (!$this->checkMinimalWordpressVersion($i))
+      $scenario->skip('Temporally skip this test because new email editor is not compatible with WP versions below 6.4');
```

Comment:
I am unsure, but don't we have a check that would fail the build if there is a skipped test in the trunk/release branch?

Reply:
@pavel-mailpoet [There](https://github.com/mailpoet/mailpoet/blob/9d94c1c5f2b0eb7c55e1ebab668bcf2536748181/mailpoet/tests/acceptance/Automation/UserRegistrationTriggerCest.php#L42) is another skipping, so it should be fine.

Reply:
Oh, I have found it. We also need those two tests to the allowed list in [CheckSkippedTestsExtension](https://github.com/mailpoet/mailpoet/blob/9a6ee9c46a15c6628c26116ddd375c88d0c6ffc6/mailpoet/tests/_support/CheckSkippedTestsExtension.php#L18-L25). Thanks @pavel-mailpoet